assets/themes/zeppelin/css/style.css

```diff
@@ -1134,7 +1195,225 @@ and (max-width: 1024px) {
   .carousel-indicators {
     margin-bottom: -60px;
   }
+
   .carousel-caption {
     padding-bottom: 60px;
   }
 }
+
+/* Advertisement Section */
+
+.advertise {
+  background: #FFF;
+  padding-bottom: 15px;
+  padding-top: 25px;
+}
+
+.advertise p {
+  padding-bottom: 18px;
+  padding-right: 9px;
+  padding-left: 9px;
+}
+
+.advertise h2 {
+  font-family: "Helvetica";
```

Comment:
2. Use thin font for `TECHNOLOGIES` and `DEPLOYMENTS`. 

- It would be nice to have less font-weight for not important texts so that users can focus on actual contents. not the title.

![image](https://cloud.githubusercontent.com/assets/4968473/26775268/1f6cfe4c-4a0f-11e7-9474-fb32167a4c8c.png)

![image](https://cloud.githubusercontent.com/assets/4968473/26775273/256ef598-4a0f-11e7-9e00-9aec76efdde3.png)